src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleDefinitionTests.ps1

```diff
@@ -128,6 +128,49 @@ function Test-RDPositiveScenarios
 
 <#
 .SYNOPSIS
+Tests verify roledefinition update with interchanged assignablescopes.
+#>
+function Test-RDUpdate
+{
+    # Setup
+    Add-Type -Path ".\\Microsoft.Azure.Commands.Resources.dll"
```

Comment:
Why are you adding the cmlets this way?  They should be added through your test controller (and likely already are)